widgets/styles/partials/_swiper-expanded.scss

```diff
@@ -0,0 +1,15 @@
+$nosto-container-margin: 20px;
```

Comment:
Rename this to match the component name (_expanded-swiper.scss)

Reply:
Resolved. Thanks!